modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/exec/exp/RexImpTable.java

```diff
@@ -3077,6 +3077,19 @@ private static class BinaryImplementor extends AbstractRexCallImplementor {
         argValueList = FlatLists.append(argValueList, fieldComparator);
       }
 
+      if (type0 == BigDecimal.class && type1 == BigDecimal.class && op == DIVIDE) {
```

Comment:
please mention the changes in comments for the class

Reply:
Done